package.json

```diff
@@ -80,6 +79,8 @@
     "reactstrap": "^8.0.0"
   },
   "resolutions": {
-    "danger/gitlab": "~6.0.0"
+    "danger/gitlab": "~6.0.0",
+    "fp-ts": "1.12.0",
```

Comment:
the error on typing `BackendClient` comes from the fact that we include different version of fp-ts package. you can verify this eecuting

`yarn why fp-ts`
```
$ yarn why fp-ts
yarn why v1.19.1
[1/4] Why do we have the module "fp-ts"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "**fp-ts@1.12.0**"
info Has been hoisted to "fp-ts"
info Reasons this module exists
   - Specified in "dependencies"
   - Hoisted from "italia-ts-commons#fp-ts"
=> Found "io-ts#**fp-ts@1.19.5**"
info This module exists because "io-ts" depends on it.
Done in 1.02s.
```

I'm not sure that the pinned version here (1.12.0) is the right one as io-ts requires 1.19.5 (try to change this above in package.json as well)